packages/derive/src/lib.rs

```diff
@@ -287,6 +334,45 @@ mod test {
         assert_eq!(actual.to_string(), expected.to_string());
     }
 
+    #[test]
+    fn contract_migrate_version_with_const_expansion() {
+        let code = quote! {
+            #[migrate_version(CONTRACT_VERSION = 66)]
+            fn migrate(deps: DepsMut, env: Env, msg: MigrateMsg) -> Response {
+                // Logic here
+            }
+        };
+
+        let actual = entry_point_impl(TokenStream::new(), code);
+        let expected = quote! {
+            #[allow(unused)]
+            #[doc(hidden)]
+            #[cfg(target_arch = "wasm32")]
+            #[link_section = "cw_migrate_version"]
+            /// This is an internal constant exported as a custom section denoting the contract migrate version.
+            /// The format and even the existence of this value is an implementation detail, DO NOT RELY ON THIS!
+            static __CW_MIGRATE_VERSION: [u8; 2usize] = *b"66";
+
+            #[allow(unused)]
+            #[doc(hidden)]
+            const CONTRACT_VERSION: u64 = 66u64;
```

Comment:
Should that be a public const?

Reply:
I think we can make it public, since it will only be available when the entrypoint is enabled, so the crate cannot really be used as a library in another contract anyways.
And that way you can use it throughout the contract's tests.